app/controllers/catalog_controller.rb

```diff
@@ -523,10 +524,7 @@ class CatalogController < ApplicationController
   # when more than one document's citation is being displayed.
   def citation
     @documents = search_service.fetch(Array(params[:id]))
-    return unless @documents.size > 1
-
-    oclc_numbers = @documents.filter_map { |document| document.oclc_number.presence }
-    @oclc_citations = Citations::OclcCitation.new(oclc_numbers:).citations_by_oclc_number
+    nil unless @documents.size > 1
```

Comment:
```suggestion
```

Is this a no-op, or something extremely clever?

Reply:
It's just a no-op.